doc.css

```diff
@@ -1,63 +1,35 @@
-/*Header*/
-.top-nav {
-  display: flex;
-  flex-wrap: wrap;
-  min-width: 100%;
-  background-color: var(--secondary-color);
-  color: var(--sec-text);
-  padding: 0.8rem;
-}
-
-.nav-right {
-  margin-left: auto;
-}
-
-/* navigation */
-.navigation {
-  display: flex;
-  font-family: system-ui;
-}
-
-.navigation .nav-brand {
-  font-weight: bold;
-  font-size: 1.1rem;
-  color: var(--sec-text);
-}
-
-.navigation .link {
-  color: var(--sec-text);
-}
-
-.right {
-  text-align: right;
-}
-
-.navigation .link-active {
-  font-weight: bold;
-}
-
 /* hero*/
-
 .sec-hero {
-  display: flex;
+  display: grid;
+  grid-template-columns: 50% 50%;
   padding: 2rem;
   gap: 5px;
-  margin: auto;
+  margin: 5rem auto;
 }
 
 .sec-hero .hero-img {
-  max-width: 350px;
   display: block;
   margin: auto;
 }
 
+.sec-hero .content-box {
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
 .sec-hero .hero-heading {
   text-align: left;
-  padding-top: 1rem;
```

Comment:
You added a padding here before but not in the new files.

Reply:
thanks for pointing it out Jia, it was no longer required because I've added padding in default h1 tags